CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.21.1 / 2017-12-12
```

Comment:
The change feels a bit bigger than a patch version. I would have made it a new minor version but that is just my opinion.